code/modules/projectiles/guns/energy/plasma.dm

```diff
@@ -24,6 +24,7 @@
 		list(mode_name="Melt", mode_desc="A much more potent plasma round for breaching tough opponents' hides", projectile_type=/obj/item/projectile/plasma, fire_sound='sound/weapons/Laser.ogg', fire_delay=12, charge_cost=25, icon="kill", projectile_color = "#FF0000"),
 		list(mode_name="INCINERATE", mode_desc="An armor-stripping plasma round", projectile_type=/obj/item/projectile/plasma/heavy, fire_sound='sound/weapons/pulse.ogg', fire_delay=14, charge_cost=30, icon="destroy", projectile_color = "#FFFFFF"),
 	)
+	wield_delay = 0.3
```

Comment:
what unit is this in?

Reply:
Centiseconds?
![image](https://user-images.githubusercontent.com/65828539/135426196-cc0eb218-38f3-4b65-a977-51e8b6276730.png)

Reply:
fastest plasma rifle in the west

Reply:
thanks for the catch, I meant for three deciseconds.